video/out/gpu/video.c

```diff
@@ -1092,7 +1092,6 @@ static void pass_info_reset(struct gl_video *p, bool is_redraw)
 
     for (int i = 0; i < VO_PASS_PERF_MAX; i++) {
```

Comment:
Could also reduce iteration count, if you move `p->pass_idx = 0;` after loop.
```suggestion
    for (int i = 0; i <= p->pass_idx; i++) {
```
tho, up to you if you want to dig into it more.